Tasks/JavaToolInstallerV0/chocoJavaInstall.ps1

```diff
@@ -0,0 +1,89 @@
+param (
+	[string]$VersionSpec
+    )
+
+# Set new machine path 
+function Set-MachinePath{
+    [CmdletBinding()]
+    param(
+        [string]$NewPath
+    )
+    Set-ItemProperty -Path 'Registry::HKEY_LOCAL_MACHINE\System\CurrentControlSet\Control\Session Manager\Environment' -Name Path -Value $NewPath
```

Comment:
You can use `GetEnvironmentVariable(…, EnvironmentVariableTarget.Machine)` and `SetEnvironmentVariable(…, ..., EnvironmentVariableTarget.Machine)` to read machine environment variables in C# and Powershell.

Reply:
Looks like this is dead code.